### ISSUE TITLE
🚀 Feature Request: 

### ISSUE THREAD

**Issue by drawcard** [elementor/elementor#18704]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Full width stretch is not available for the new container element.

### Describe the solution you'd like

It would be nice if the [full width page stretch feature](https://elementor.com/blog/v09-think-wider/) is re-introduced for containers, so parent containers can stretch across the screen regardless of boundaries. 

### Describe alternatives you've considered

Manual JS coding, not a nice experience for new users though.

**Comment by Daveden2** [comment]
Unless I'm missing something. Fullwidth already exists for containers. Just go to Edit Container and change Content Width from Boxed to Full Width

![image](https://user-images.githubusercontent.com/105490669/169720022-ea48f0f9-b306-4a56-ae59-1df134e04adc.png)

**Comment by drawcard** [comment]
@Daveden2 thanks, but unfortunately that doesn't work - it just matches the width of the parent container.

**Comment by Daveden2** [comment]
> @Daveden2 thanks, but unfortunately that doesn't work - it just matches the width of the parent container.

That is exactly how full width (stretch) works. It matches the (% width you set) for its content to width of its parent container. So, if the parent container is the body,  and you set the width to 100%, then it'll stretch to the full width of the body. If the parent container is another container, then it'll stretch to the full width of that container. 

 In your case, simply set the parent container to full width (100%).

**Comment by drawcard** [comment]
However, in my case I don't want to set the parent container to 100% width, because my parent container is the page template itself and other content relies on it being a boxed width.

The earlier solution that used JS to let child containers ignore parents, is the solution I'm after.

**Comment by Daveden2** [comment]
Oh yeah. I see what you mean now.  My apologies for the misunderstanding. 

Maybe you can try to update your question with screenshots to show specific use cases and how it'll benefit most users.

**Comment by drawcard** [comment]
@rami-elementor is there any possibility of bringing back stretch container functionality to the Containers?

This is the functionality I'd like to see return: https://github.com/elementor/elementor/blob/2cd3d50d8d87edde9cd3c820f5a0f8b1f2c4f24f/assets/dev/js/frontend/tools/stretch-element.js#L18

I've also tried the tricks in this article but can't seem to get them working: https://css-tricks.com/full-width-containers-limited-width-parents/

**Comment by acal** [comment]
I agree there is a need for the combination of using page templates with the content using boxed width but where arbitary sections can be overridden as was the case with old full-width page stretch feature. It was very useful for layouts with user-defined full-width background color changes for arbitrary sections. I haven't found a user-friendly way, like it was before flex containers, yet.

**Comment by JonasHaas** [comment]
this is still a needed feature!

**Comment by Maximillion195** [comment]
Yes, this was a really useful feature. I'm having to use a mix of Sections and Containers to make this work at the moment.

**Comment by bodhi1970** [comment]
I can't believe this is still an unsolved issue. Make containers stretch please.

**Comment by vikecall** [comment]
I found a solution:
https://elementor.com/help/full-width-not-working/
it's your theme.  
My theme has an option: Page Builder Mode, making this active, solved the problem.

**Comment by massikage** [comment]
Thank you my friend

**Comment by yorkfrog** [comment]
@drawcard thanks for the CSS, worked nearly perfectly.  I had to adjust slightly else I got a gap on the right margin as another rule was hijacking max-width.

`.full-width {
        left: 50%;
	margin-left: -50vw;
	margin-right: -50vw;
	max-width: 100vw !important;
	position: relative;
	right: 50%;
	width: 100vw;
}`

**Comment by sofyan784** [comment]
solution fix : from site setting < layout < padding left and right to 0, top and bottom to 10 or 20

**Comment by davidamarek** [comment]
This still isn't working for me - I want to stretch the container so that a background colour is full width, or an image stretches to full width - still not possible?

**Comment by nexgencon** [comment]
![image](https://github.com/elementor/elementor/assets/151937346/09eaa010-a182-450c-936d-f6b8b6a66d03)
go to customize < page < container layout, change to full width. Then the whole site will go with full width. If you want several container go with 80 vw, You should adjust each container width manually.

**Comment by EarlBadger** [comment]
I’ve just stumbled into this problem converting a 5 site multisite that uses multiple single page templates with constrained content and full-width backgrounds. I’ve tried all of the fixes suggested here and only the CSS works for me - using Hello theme. However, it’s a really time waster having to add the CSS Class to all of the relevant sections. Fine if you’re building from scratch but painful if converting existing.

The suggested stretch fix would be hugely appreciated as I expect we’ll be converting more sites in the future.

**Comment by ioweb-gr** [comment]
The problem is that sections used to do this regardless of other settings, yet containers lose this functionality. E.g I have a section that stretches in full width regardless of the layout. When I click on convert, and it's converted to container, I would expect it to have the same width as the original section, but it doesn't and can't make it happen.

**Comment by EarlBadger** [comment]
Since I first came across this issue five days ago something has changed. Either it was the release of 3.19 or I'm so far down this particular rabbit hole I missed a turning the first time around. I can now get the background to full width (setting the padding of the container of the Post Content widget in the template to zero, whilst still in Boxed mode). However when I try and add 20px padding left and right to the same container (so the content on pages sits away from the window edge in responsive modes) it moves the container away from the edge - effectively the padding is doing exactly the same as if I set the left and right margin to 20px. So I'd have to set the left and right padding in each container on each page I create...thus negating the benefit of using templates.

I submitted a ticket to Elementor support on this and was passed to Tier 2 support. After a bit of confusion about what exactly the problem was (they couldn't replicate) I sent them a video explanation. I pretty quickly got a response, "I can see the issue and this problem is already being handled by our developers. I am not able to provide you with ETA of the resolution of this issue". 

So there is a recognised issue and a fix is being worked on. I agree with drawcard, it seems that in the move to containers the JS stretch toggle was dropped or missed.

**Comment by themeland** [comment]
This is a major issue when creating a page with Default Template. Most of the cases, we use Elementor Default Template & also sometimes we need to stretch the container to make it full width.

I have some markup on the body for the Default Template which contains the Header, Breadcrumb & Footer. When I want to add a full width container, it's not overlapping to full width for the markup. Previously, when Elementor Section has a Stretch option, I simply used that to make this a full width. But I'm unable to do it. The container should have the option of "Stretch" to have this feature.

**Comment by ciaojeps** [comment]
I'm not sure if a solution has already been found, but this works for me:
`.full-width {
  position: relative;
  width: 100vw!important; /* Sets the width to 100% of the viewport width */
  left: calc(50% - 50vw); /* Centers the element by adjusting its left position */
  max-width: inherit!important; /* Ensures the element can inherit the maximum width from its parent */
}`

Obviously, the full-width class must be assigned to the flexbox container.

**Comment by snsnsjsn** [comment]
I found this option from woodmart will work perfect to ignore elementor set , and no need to set page layout to full width .

[XTemos]Layout
see the pictures

any container or grid there will be this option to set elementor full with

<img width="1569" height="824" alt="Image" src="https://github.com/user-attachments/assets/aa3df895-63eb-4170-9aef-6c67458fa478" />

<img width="1413" height="879" alt="Image" src="https://github.com/user-attachments/assets/51131c4b-a5e7-453a-a592-ba96568558be" />

but this will produce new problem, like I could not set the grid 6 grids on pc and responsive to 2 grid on mobile phone .

**Comment by rockythemes** [comment]
I know this is overkill, but this is needed for instance on themes that are using a Boxed layout - my theme has this type of layout with a determined width for the main div, where all the Elementor elements are added, so in this case, would anyone be interested in having a tiny plugin created just for this? I can probably make it in 10 minutes or so, if there's enough interest for it. Maybe I can add the Custom CSS and Attributes for Container as well - this will work for top level container, not inner containers. Let me know if there's any interest in that. Thanks.
